Color 3D scene

soulilaigle opened the pull request:
multicolor_front
    
            added :
                    - colors is now beautiful on
                    the 3d scene
    
            modified :
                    - background is now in
                    the glb file
            to do : 
                    - modify background for better render

soulilaigle committed:
colorised_front

Added:
	- 3 brand new colors
	- Color choice of the front end view
	- Go in Settings -> General and then choose

soulilaigle committed:
multicolor_front

	added :
		- colors is now beautiful on
		the 3d scene

	modified :
		- background is now in
		the glb file

Dyamen1411 reviewed (changes requested):
Indentation is wrong :(

soulilaigle committed:
color_map

	edited :
		- better indentation and format

Dyamen1411 committed:
Made color change persistant in the browser cache.

Changed:
	- The website theme (aka color) is now
		stored within the
		browser cache (localStorage)
		to make it persistant across
		sessions.
